ProfileAPI/ProfileAPI/Controllers/ProfileController.cs

```diff
@@ -0,0 +1,100 @@
+﻿using System;
+using System.Collections.Generic;
+using Microsoft.AspNetCore.Mvc;
+using ProfileAPI.Models;
+using ProfileAPI.Services;
+
+namespace ProfileAPI.Controllers
+{
+    [Route("[controller]")]
+    [ApiController]
+    public class ProfileController: ControllerBase
+    {
+        private readonly ProfileService _profileService;
```

Comment:
The convention we chose for the ConferenceService is camelCase without underscores for private variables and PascalCase for public variables. Would be nice to be consistent for the same language.